content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -75,59 +145,169 @@ Based on resources the task has used, Datadog assigns the appropriate portion of
-- Cluster idle: Cost of provisioned IOPS, throughput, or storage for volumes not claimed by any pods that day.
+| Spend type | Description    |
+| -----------| -----------    |
+| Usage | Cost of provisioned IOPS, throughput, or storage used by workloads. Storage cost is based on the maximum amount of volume storage used that day, while IOPS and throughput costs are based on the average amount of volume storage used that day. |
+| Workload idle | Cost of provisioned IOPS, throughput, or storage that are reserved and allocated but not used by workloads. Storage cost is based on the maximum amount of volume storage used that day, while IOPS and throughput costs are based on the average amount of volume storage used that day. This is the difference between the total resources requested and the average usage. **Note:** This tag is only available if you have enabled `Resource Collection` in your [**AWS Integration**][101]. To prevent being charged for `Cloud Security Posture Management`, ensure that during the `Resource Collection` setup, the `Cloud Security Posture Management` box is unchecked. |
+| Cluster idle | Cost of provisioned IOPS, throughput, or storage that are not reserved by any pods that day. This is the difference between the total cost of the resources and what is allocated to workloads. |
 
 **Note**: Persistent volume allocation is only supported in Kubernetes clusters, and is only available for pods that are part of a Kubernetes StatefulSet.
 
+[101]: https://app.datadoghq.com/integrations/amazon-web-services
+
+{{% /tab %}}
+{{% tab "Google" %}}
+
+### Compute
+
+The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
+
+Costs are allocated into the following spend types:
+
+| Spend type | Description    |
+| -----------| -----------    |
+| Managed service fee | Cost of associated fees charged by the provider for managing the cluster, such as GKE fees. |
+| Usage | Cost of resources (such as memory and CPU) used by workloads, based on the average usage on that day. |
+| Workload idle | Cost of resources (such as memory and CPU) that are reserved and allocated but not used by workloads. This is the difference between the total resources requested and the average usage. |
+| Cluster idle | Cost of resources (such as memory and CPU) that are not reserved by workloads in a cluster. This is the difference between the total cost of the resources and what is allocated to workloads. |
```

Comment:
```suggestion
| Cluster idle | Cost of resources (such as memory and CPU) that are not reserved by workloads in a cluster. This is the difference between the total cost of the resources and what is allocated to workloads. |
| Not monitored | Cost of resources where the spend type is unknown. Installing the datadog agent on these clusters/nodes will resolve the issue |
```
For GCP only: we need a `Not monitored` section that pretty much tells users that they need to install the datadog agent to get the full support here. I am not sure about my wording though. Feel free to update it